### PR TITLE
Home page 13

### PR DESCRIPTION
**Pull request by hannahli23**
- Created an Accordion Styled drop down 
- Added a Navlink to lead to the maps page
- Added the semantic-ui-react stylesheet

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -13,6 +13,11 @@ import packageJson from '../package.json';
 
 global.appVersion = packageJson.version;
 
+const styleLink = document.createElement("link");
+styleLink.rel = "stylesheet";
+styleLink.href = "https://cdn.jsdelivr.net/npm/semantic-ui/dist/semantic.min.css";
+document.head.appendChild(styleLink);
+
```
Comment (oamyette3) [inline client/src/index.js]: Put css in a different file and link!!

**Review by oamyette3 — DISMISSED** [review]

File: client/package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-scripts": "3.3.0",
-    "semantic-ui-react": "^0.88.2"
+    "semantic-ui-react": "latest"
```
Comment (evan10s) [inline client/package.json]: Probably shouldn't target `latest` version of a dependency, for consistency reasons.  You can put in the specific version you need

**Review by evan10s — CHANGES_REQUESTED** [review]

**Comment by evan10s** [comment]
Please also increment the version in client/package.json since you modified files there

**Review by evan10s — CHANGES_REQUESTED** [review]
Fix the package.json dependency set to "latest" and remove the raw Javascript to add CSS

**Review by evan10s — APPROVED** [review]
lgtm

**Pull request merged**